crates/biome_module_graph/src/js_module_info.rs

```diff
@@ -46,6 +50,10 @@ impl JsModuleInfo {
         }
     }
 
+    pub fn as_diagnostics(&self) -> &[ModuleDiagnostic] {
```

Comment:
Nit: Using the `as_` prefix seems odd for what appears to be an ordinary getter.

```suggestion
    pub fn diagnostics(&self) -> &[ModuleDiagnostic] {
```